packages/core/src/App/Containers/Layout/header/header.tsx

```diff
@@ -2,24 +2,13 @@ import React from 'react';
 import { useLocation } from 'react-router-dom';
 import { useFeatureFlags } from '@deriv/hooks';
 import { useReadLocalStorage } from 'usehooks-ts';
-import { isDTraderV2, makeLazyLoader, moduleLoader, routes } from '@deriv/shared';
+import { makeLazyLoader, moduleLoader, routes } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 import { useDevice } from '@deriv-com/ui';
 import classNames from 'classnames';
-import DTraderV2HeaderLoader from './dtrader-v2-header-loader';
 
 const HeaderFallback = () => {
-    const location = useLocation();
-    const is_contract_details = location.pathname.startsWith('/contract/');
-    const is_positions = location.pathname === routes.trader_positions;
-
-    return (
-        <div className={classNames('header', { 'header-v2': isDTraderV2() })}>
-            {isDTraderV2() && !is_contract_details && (
-                <DTraderV2HeaderLoader show_notifications_skeleton={!is_positions} />
```

Comment:
@vinu-deriv
After the change, skeleton loader is no longer applied to the header while the page is loading:
Current master VS test link:
<img width="200" alt="Screenshot 2024-08-20 at 5 27 57 PM" src="https://github.com/user-attachments/assets/84e89e6f-dede-4b64-ad76-0e013a845e08">
<img width="200" alt="Screenshot 2024-08-20 at 5 24 04 PM" src="https://github.com/user-attachments/assets/16c7672d-b6a4-4926-affd-fedc19fc3a49">

Please confirm with designers whether or not it's expected that we won't show skeleton loader instead of the header while the page is loading as we apply the old header.🙏🏻
I suspect that this is expected because the old header structure doesn't match the existing design for the header skeleton.

Reply:
I had removed header loader, I will check with designers once

Reply:
@maryia-deriv I checked with designers, as per them we can go without loaders